clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus_test.go

```diff
@@ -33,6 +33,25 @@ import (
 	measurementutil "k8s.io/perf-tests/clusterloader2/pkg/measurement/util"
 )
 
+var (
+	// klogv1 allows users to turn on/off logging to stderr only through
+	// the use of flag. This prevents us from having control over which
+	// of the test functions have that mechanism turned off when we run
+	// go test command. There's a programmatic solution to this issue
+	// in klogv2, see https://github.com/kubernetes/klog/pull/100.
+	// TODO: Remove this hack after perf-tests is ported to klogv2.
```

Comment:
s/perf-tests/clusterloader2
I would create a separate issue for making clusterloader2 leverage klogv2. WDYT?

Reply:
Resolving - after offline discussion we decided to create #1286 to get rid of this.